src/frameworks/nuxt/index.ts

```diff
@@ -3,11 +3,14 @@ import { readFile } from "fs/promises";
 import { basename, join } from "path";
 import { gte } from "semver";
 import { BuildResult, findDependency, FrameworkType, relativeRequire, SupportLevel } from "..";
+import { warnIfCustomBuildScript } from "../utils";
 
 export const name = "Nuxt";
 export const support = SupportLevel.Experimental;
 export const type = FrameworkType.Toolchain;
 
+const NUXT_DEFAULT_BUILD_SCRIPTS = ["nuxt build"];
```

Comment:
Does this differ between nuxt 2 & 3?

Reply:
Although the docs mention the [`nuxi build`](https://nuxt.com/docs/api/commands/build) command for Nuxt 3, when a Nuxt 3 project is created with `npx nuxi init <project-name>` the `package.json` contains the `nuxt build` command, the same as in a Nuxt 2 project.